packages/sirius-web/backend/sirius-web-application/src/main/java/org/eclipse/sirius/web/application/views/details/services/PropertiesEventProcessorFactory.java

```diff
@@ -10,7 +10,7 @@
  * Contributors:
  *     Obeo - initial API and implementation
  *******************************************************************************/
-package org.eclipse.sirius.components.collaborative.forms;
```

Comment:
Maybe rename the class to `DetailsEventProcessorFactory`?

Reply:
This class will be renamed once dependencies will be moved to `sirius-web-application`